ozark/src/main/java/org/glassfish/ozark/binding/BindingResultImpl.java

```diff
@@ -89,22 +91,26 @@ public BindingError getBindingError(String param) {
     }
 
     @Override
-    public Set<ConstraintViolation<?>> getAllViolations() {
-        return violations;
+    public Set<ValidationError> getAllValidationErrors() {
+        return Collections.unmodifiableSet(validationErrors);
     }
 
     @Override
-    public Set<ConstraintViolation<?>> getViolations(String propertyPath) {
```

Comment:
Good catch. I'll update these. Seems like they are out of sync with the interface for quite some time. It was called `propertyPath` in the old days. :)